uber/migration.py

```diff
@@ -0,0 +1,87 @@
+from collections import OrderedDict
+from os.path import abspath, dirname, join
+
+from alembic import command
+from alembic.config import Config as AlembicConfig
+from alembic.script import ScriptDirectory
+
+from sideboard.internal.imports import plugin_dirs
+
+
+# Path to "alembic.ini", cached here for convenience
+alembic_ini_path = abspath(join(plugin_dirs['uber'], 'alembic.ini'))
```

Comment:
Of course, your use of ``plugin_dirs`` later on is something that couldn't be done as easily without relying on ``sideboard.internal.imports``.

I didn't think of this at the time when I was reviewing the ``sideboard.internal.imports.plugin_dirs`` field that you added this week, but I wonder if we should expose some kind of officially supported plugin registry in ``sideboard.lib`` so that we're not forced to rely on ``sideboard.internal`` for this kind of thing.  Food for thought for later, I suppose.